src/styles/index.scss

```diff
@@ -1,3 +1,152 @@
+$colorButtonBuy: #00acdc;
+
 body {
   margin: 0;
 }
+
+.content {
+  margin: 0 auto;
+  width: 700px;
+  padding-top: 30px;
+}
+
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    p {
+      margin: 0;
+    }
+
+    width: 100%;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    letter-spacing: 0;
+    text-align: left;
+
+    &--position {
+      margin-top: 40px;
+    }
+  }
+
+  &__code {
+    width: 100%;
+    font-size: 10px;
+    line-height: 14px;
+    letter-spacing: 0;
+    color: #616070;
+
+    &--position {
+      margin-top: 4px;
+    }
+  }
+
+  &__rate {
+    width: 100%;
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    .reviews {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      letter-spacing: 0;
+      text-align: right;
+      color: #060b35;
+    }
+
+    &--position {
+      margin: 16px 0 24px;
+    }
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+
+    .price {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      letter-spacing: 0;
+      text-align: left;
+      color: #616070;
+    }
+
+    .amount {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      letter-spacing: 0;
+      text-align: right;
+      color: #060b35;
+    }
+  }
+
+  &__buy {
+    width: 100%;
+    background-color: $colorButtonBuy;
+    border-radius: 5px;
+    text-transform: uppercase;
+    color: #fff;
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    &:hover {
+      box-sizing: border-box;
+      border: 1px solid $colorButtonBuy;
+      background-color: #fff;
+      color: #00acdc;
+    }
+
+    &--position {
+      margin-top: 16px;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 96px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars--4 :nth-child(-n + 4) {
```

Comment:
Consider keeping all modifiers working, not just the 4-star modifier

Reply:
done